generators/entity-server/templates/ui/widgets/tableWidget/src/components/filters/FiltersContainer.js.ejs

```diff
@@ -31,10 +31,24 @@ const styles = {
 };
 
 const FiltersContainer = props => {
-  const { classes, filters, applyFilter, update, remove, clear, add, t } = props;
+  const { classes, filters, applyFilter, update, remove, clear, add, t, error } = props;
 
   const hasFilters = filters.length > 0;
 
+  const handleApplyFilter = () => {
+    const errors = hasFilters && filters.map((filt) => {
+      if (filt.field && filt.operator && ['specified', 'unspecified'].indexOf(filt.operator) === -1) {
```

Comment:
- where the `specified` and `unspecified` constants come from?

- what about using the `includes` method instead of the good old indexOf(str) !== -1 check?

Reply:
there wasn't any said constants made so I added them in utils helper. Also I changed the indexOf method to includes (OMG i guess I'm old haha :P)